VATSpy.dat

```diff
@@ -18009,7 +18009,8 @@ LJLA|Ljubljana||LJLA
 LKAA|Praha||LKAA
 LLLL|Tel Aviv|LLTA|LLLL
 LLLL|Tel Aviv|SHOLET|LLLL
-LLSC|Tel Aviv South Control||LLSC
+LLSC|Tel Aviv South||LLSC
+LLPT|Tel Aviv North|PLUTO|LLPT
 ; dummy codes for Israel CVFR centers
 LLPL|Pluto|PLUTO|LLLL

```

Comment:
2 entries with Pluto as logon, should only be 1

Reply:
@prryplatypus can you check with the guy?

Reply:
From the request that we received it does look like this change is correct. I'm going to merge it in and if not we can fix it in a later version.